client/src/components/pages/NeowisePage.js

```diff
@@ -19,6 +19,7 @@ function NeowisePage() {
   const [modalOpen, setModalOpen] = useState(false);
   const [nasaItems, setNasaItems] = useState(null);
   const [search, setSearch] = useState("");
+  const [lazyButton, setLazyButton] = useState(false);
```

Comment:
You could specify the naming:
`showLazyButton`

Otherwise, it's unclear what this is used for :)